src/ClientTlsContext.php

```diff
@@ -443,13 +459,16 @@ public function toStreamCryptoMethod(): int
     {
         switch ($this->minVersion) {
             case self::TLSv1_0:
-                return self::TLSv1_0 | self::TLSv1_1 | self::TLSv1_2;
+                return self::TLSv1_0 | self::TLSv1_1 | self::TLSv1_2 | self::TLSv1_3;
```

Comment:
Should TLS 1.3 only be enabled on PHP 7.4+?

Reply:
I was hoping our CI would show if the value was just ignored below 7.4. Do you know? I'm otherwise fine limiting it to 7.4+.

Reply:
With OpenSSL >= 1.1 it should definitely work: https://github.com/php/php-src/blob/53ea910d1760c87b6110a461f13ebe0e244c9914/ext/openssl/xp_ssl.c#L1721-L1724

Otherwise seems not: https://github.com/php/php-src/blob/53ea910d1760c87b6110a461f13ebe0e244c9914/ext/openssl/xp_ssl.c#L1628

Reply:
But no harm in the flag being set, just ignored?

Reply:
IIRC it might cause misbehavior, but I wasn't able to find anything on bugs.php.net.

Reply:
Well, it's not hard to remove remove the flag for versions under 7.4, so I'll just do that to be safe.